### DISCUSSION TITLE
How to generate outputs from the PPOTrainer of chatgpt?

### DISCUSSION THREAD

**Discussion by huliangbing** [hpcaitech/ColossalAI#2906] [Community | Q&A]
How to generate outputs from the PPOTrainer of chatgpt? Can we generate outputs from reward_model or initial_model?

Can you show me the code like this:

tokenizer = AutoTokenizer.from_pretrained(checkpoint)
inputs = tokenizer(prompt, return_tensors="pt")
model = AutoModelForSeq2SeqLM.from_pretrained(checkpoint)
outputs = **### model.generate**(**inputs, num_beams=5, num_beam_groups=5, max_new_tokens=30)
tokenizer.decode(outputs[0], skip_special_tokens=True)

Thanks!

**Comment by ht-zhou** [comment]
Thanks for your feedback. We have already supported actor-inference in our newly updated PR.

**Reply by huliangbing** [reply]
Thanks very much!

**Comment by taishiciR** [comment]
how to infer a RM(reward_model)  like rm_checkpoint.pt?
